packages/ng-table-builder/src/components/table-thead/table-thead.component.ts

```diff
@@ -33,9 +33,18 @@ export class TableTheadComponent<T> {
 
     constructor(protected readonly filterable: FilterableService<T>) {}
 
+    public sortIfEnabled(): void {
+        const key: string | null | undefined = this.columnSchema?.key;
+        if (isNotNil(key) && (isTrue(this.columnSchema?.sortable) || isNotNil(this.sortableDefinition[key]))) {
+            this.sortByKey.emit(key);
+        }
+    }
+
     public openFilter(key: string | null | undefined, event: MouseEvent): void {
         if (isNotNil(key)) {
             this.filterable.openFilter(key, event);
         }
+        event.stopPropagation();
```

Comment:
please, add comment about why we move current code here

```ts
/**
 Note: .... 
*/
event.stopPropagation();
event.preventDefault();
```

Reply:
i think no need to explain why it moved here from `FilterableService`. i comment the reason to call it at all.